lib/embedded_ansible.rb

```diff
@@ -11,13 +17,114 @@ def self.enabled?
   end
 
   def self.running?
-    # TODO
-    false
+    services.map { |service| LinuxAdmin::Service.new(service).running? }.inject(:&)
+  end
+
+  def self.configure
+    configure_secret_key
+    with_inventory_file do |inventory_file_path|
+      setup_params = {
+        :e => "minimum_var_space=0",
+        :k => "packages,migrations,supervisor",
```

Comment:
Would it make sense to put this list into a explaining constant/variable/method so it's obvious why one has `supervisor` in the list and the one below doesn't?